### PR TITLE
Replace fail+catch pattern with assertThatThrownBy in trino-phoenix5

### PR DESCRIPTION
**Pull request by choiwaiyiu**
Relates to https://github.com/trinodb/trino/issues/6553

### CODE REVIEW

File: plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java [hunk 1]
```diff
@@ -198,12 +198,9 @@ public void testSchemaOperations()
         assertUpdate("CREATE SCHEMA new_schema");
         assertUpdate("CREATE TABLE new_schema.test (x bigint)");
 
-        try {
-            getQueryRunner().execute("DROP SCHEMA new_schema");
-            fail("Should not be able to drop non-empty schema");
-        }
-        catch (RuntimeException e) {
-        }
+        assertThatThrownBy(() -> getQueryRunner().execute("DROP SCHEMA new_schema"))
```
Comment (kokosing) [inline plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java:201]: Can you please also fix `io.trino.plugin.phoenix.TestPhoenixConnectorTest#testSchemaOperations`? It is a copy.

**Review by kokosing — COMMENTED** [review]
looks good % single comment. Thanks.

File: plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java (hunk 1 above)
Comment (choiwaiyiu) [reply plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java:201]: @kokosing 
`io.trino.plugin.phoenix.TestPhoenixConnectorTest#testSchemaOperations` has been fixed in https://github.com/trinodb/trino/pull/6767.

https://github.com/trinodb/trino/blob/master/plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixConnectorTest.java#L195-L207

**Review by kokosing — APPROVED** [review]

**Review by kokosing — APPROVED** [review]

**Comment by kokosing** [comment]
Merged, thanks!

**Pull request merged**